lib/oper1.g

```diff
@@ -360,6 +360,40 @@ BIND_GLOBAL( "InstallOtherMethod",
     end );
 
 
+#############################################################################
+##
+#F  InstallEarlyMethod( <opr>,<method> )
+##
+##  <#GAPDoc Label="InstallEarlyMethod">
+##  <ManSection>
+##  <Func Name="InstallEarlyMethod" Arg="opr,method"/>
+##
+##  <Description>
+##  installs a special "early" function method <A>method</A> for the
+##  operation <A>opr</A>. An early method is special in that it bypasses
+##  method dispatch, and is always the first method to be called when
+##  invoking the operation. This can be used to avoid method selection
+##  overhead for certain special cases.
```

Comment:
I wondered if we should add something to clarify that this operation is called even if the arguments to not satisfy the argument filters for the method (I realise technically there was never an explicit check for these, they are enforced in `InstallMethod`, but I thought it might be worth clarifying.

Reply:
I've now added some text for this.